likelihood/cl_like/cl_like.py

```diff
@@ -206,10 +206,12 @@ def _get_bz(self, cosmo, name, **pars):
         """ Get linear galaxy bias. Unless we're using a linear bias,
         this should be just 1."""
         z = self.bin_properties[name]['z_fid']
+        zmean = self.bin_properties[name]['zmean_fid']
         bz = np.ones_like(z)
         if self.bz_model == 'Linear':
             b0 = pars[self.input_params_prefix + '_' + name + '_b0']
-            bz *= b0
+            bp = pars[self.input_params_prefix + '_' + name + '_bp']
+            bz *= b0 + bp * (z - zmean)
```

Comment:
Since multiplying by `(z-zmean)` already changes the full array I'd just do `bz = ...` instead of `bz *= ...`